acceptance/localcluster/docker.go

```diff
@@ -146,6 +146,12 @@ func maybePanic(err error) {
 // Remove removes the container from docker. It is an error to remove a running
 // container.
 func (c *Container) Remove() error {
+	if os.Getenv("CIRCLE_ARTIFACTS") != "" {
```

Comment:
`os.Getenv("CIRCLECI") == "true"` looks like the official way to test for this (https://circleci.com/docs/environment-variables)